api/feedbooks.py

```diff
@@ -0,0 +1,457 @@
+import datetime
+import feedparser
+from nose.tools import set_trace
+from StringIO import StringIO
+from zipfile import ZipFile
+from lxml import etree
+import os
+from flask_babel import lazy_gettext as _
+
+from core.opds import OPDSFeed
+from core.opds_import import (
+    OPDSImporter,
+    OPDSImportMonitor,
+    OPDSXMLParser,
+)
+from core.model import (
+    DataSource,
+    ExternalIntegration,
+    Hyperlink,
+    Resource,
+    Representation,
+    RightsStatus,
+)
+from core.util.epub import EpubAccessor
+
+
+class FeedbooksOPDSImporter(OPDSImporter):
+
+    REALLY_IMPORT_KEY = 'really_import'
+    LANGUAGE_KEY = 'language'
+    REPLACEMENT_CSS_KEY = 'replacement_css'
+
+    NAME = ExternalIntegration.FEEDBOOKS
+    DESCRIPTION = _("Import open-access books from FeedBooks.")
+    SETTINGS = [
+        {
+            "key": REALLY_IMPORT_KEY,
+            "type": "select",
+            "label": _("Really?"),
+            "description": _("Most libraries are better off importing free Feedbooks titles via an OPDS Import integration from NYPL's open-access content server or DPLA's Open Bookshelf. This setting makes sure you didn't create this collection by accident and really want to import directly from Feedbooks."),
+            "options": [
+                { "key": "false", "label": _("Don't actually import directly from Feedbooks.") },
+                { "key": "true", "label": _("I know what I'm doing; import directly from Feedbooks.") },
+          ],
+          "default": "false"
+        },
+        {
+            "key": LANGUAGE_KEY,
```

Comment:
In content, the Feedbooks import script created a collection for each language. Now users can create whichever individual language collections they want on their site.